.coafile

```diff
@@ -29,12 +20,13 @@ language = Python
 remove_all_unused_imports = true
 
 [autopep8]
-bears = PEP8Bear, PycodestyleBear
+bears += PycodestyleBear
```

Comment:
why are you making changes here?

Reply:
this issue doesn't involve making changes to `.coafile`.

Reply:
@kwsohn your first commit was fine, you were supposed to correct the errors given by @gitmate-bot in your code. Do not change the `coafile`.

Reply:
Okay, sure thing. I will undo my changes to the coafile^